tests/kernel/mem_protect/mem_map/testcase.yaml

```diff
@@ -7,7 +7,9 @@ tests:
   kernel.memory_protection.mem_map:
     filter: CONFIG_MMU and not CONFIG_X86_64
     extra_sections: _TRANSPLANTED_FUNC
-    extra_args: CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
+    extra_configs:
+      - CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
+      - CONFIG_CBPRINTF_REDUCED_INTEGRAL=y
```

Comment:
I think this section applies to all boards. It is possible better to create a board overlay just for `qemu_x86_tiny`.

Reply:
Yes, but it's safe to use this on all targets as the test doesn't require long long cbprintf support.